src/RamRelation.h

```diff
@@ -108,6 +113,11 @@ class RamRelation : public RamNode {
         return arity;
     }
 
+    /* @brief Get arity of relation */
```

Comment:
Is this comment true? From the name I would guess it gets the number of heights of something.

Reply:
This comment is indeed wrong, fixed in clean up commit.